scripts/entrypoint.sh

```diff
@@ -13,6 +13,10 @@ mkdir -p /home/odkuser
 chown odkuser:odkuser /home/odkuser
 [ -d /home/odkuser/.data ] && chown odkuser:odkuser /home/odkuser/.data
 [ -d /home/odkuser/.data/oaklib ] && chown odkuser:odkuser /home/odkuser/.data/oaklib
+if [ ! -d /tools/.coursier-cache ] ; then
+    mkdir /tools/.coursier-cache
+    chown odkuser:odkuser /tools/.coursier-cache
+fi
```

Comment:
You removed Ammonite, but build logic to handle coursier cache? Is `/tools/.coursier-cache` used for anything other than `ammonite`?

Reply:
Scala-CLI uses it as well. :)